jctools-core/src/main/java/org/jctools/queues/MpscArrayQueue.java

```diff
@@ -103,6 +105,7 @@ public MpscArrayQueueL2Pad(int capacity) {
     }
 }
 
+//$gen:ordered-fields
```

Comment:
This tells the generator to strip all the static initialisers and fields from this class and replace them with atomic field updaters. It also tells them to replace the body of each method that looks like an ordered operation on a field with appropriate accessors on the atomic field updaters.